### ISSUE TITLE
Issue with client vs server configs

### ISSUE THREAD

**Issue by milonti** [xJon/The-1.12.2-Pack#272]
Hello, I've been running a 1.12.2 multiplayer server for a couple months off of akliz.gg.net, and every time the modpack updates, I have to redo config files for Quark. For some reason, the installation of configs on the server don't match the client-side config files. 

Normally this wouldn't be too much of an issue, but it seems my server installation keeps turning on blocks in the Quark config (limestone, slate, etc) and this make the server allocate ids to them. When I connect (or any of my other players) without the same blocks turned on in our configs, we crash out with the "Fatal Missing Registries" error. 

After a few debugging sessions I managed to figure out the problem and fix it (like I said, mismatched configs). But the fact it came back a patch later was... surprising. So I was submitting this issue to ask if it was a problem inherent to the modpack (an oversight like this is small and easy to make and easy to have hidden for awhile) or if I should go ask my server provider about their install process. 

TLDR: Can you double check config files that assign/register block ids match between your server pack and your client pack? Thank you so much and I've been enjoying your work a BUNCH!!!

**Comment by xJon** [comment]
Hey, thank you for reporting!
There was indeed one change between the configs, but it was something that effects the client only, so I didn't think there would be any issues.
The configs in the available server packages now match perfectly, so hopefully that will fix the issues.
